adapter/persistence/src/main/java/org/anastaklo/portal/adapter/persistence/CalendarJpaEntity.java

```diff
@@ -0,0 +1,38 @@
+package org.anastaklo.portal.adapter.persistence;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+@Entity
+@Table(name = "calendar")
+public class CalendarJpaEntity extends BaseJpaEntity{
+    
+    @OneToMany(mappedBy = "parentCalendar")
+    @Getter
+    @Setter
+    private List<CalendarJpaEntity> childCalendars = new ArrayList<>();
```

Comment:
why should it know about children?

Reply:
honestly I don't know :) we need to discuss calendar specific part (maybe in another task, or I can remove this mapping if it's definitely unnecessary)